### ISSUE TITLE
Master Issue for model bringup from Torch and Jax

### ISSUE THREAD

**Issue by tapspatel** [tenstorrent/tt-mlir#1705]
Torch
1. Write a model in torch
2. Compile using torch.compile

- Showcase torch backend and custom_backend
- Showcase FX graph generated

3. For custom_backend

- Showcase stablehlo IR (mlir)
- Showcase TT IR (mlir)
- Showcase TTNN IR (mlir)

4. In visualizer, have different run options

- From stablehlo IR
- From TT IR
- From TTNN IR

5. In visualizer, showcase golden accuracy information

- For stablehlo IR
- For TT IR
- For TTNN IR

6. In visualizer, showcase performance information

- For stablehlo IR
- For TT IR
- For TTNN IR

7. In visualizer, showcase memory information

- For stablehlo IR
- For TT IR
- For TTNN IR

JAX
1. Write a model in jax
2. Compile using jax.jit

- Showcase vhlo IR (mlir)
- Showcase shlo IR (mlir)
- Showcase TT IR (mlir)
- Showcase TTNN IR (mlir)

3. In visualizer, have different run options

- From vhlo IR
- From shlo IR
- From TT IR
- From TTNN IR

4. In visualizer, showcase golden accuracy information

- For vhlo IR
- For stablehlo IR
- For TT IR
- For TTNN IR

5. In visualizer, showcase performance information

- For vhlo IR
- For stablehlo IR
- For TT IR
- For TTNN IR

6. In visualizer, showcase memory information

- For vhlo IR
- For stablehlo IR
- For TT IR
- For TTNN IR

Sub-Issues to finish goals
MLIR
- [x] https://github.com/tenstorrent/tt-mlir/issues/1683
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1706
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1707
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1708
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1709
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1710
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1711
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1712
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1713
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1714
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1715
- [x] https://github.com/tenstorrent/tt-mlir/issues/1719
- [x] https://github.com/tenstorrent/tt-mlir/issues/1720
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1721
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1722
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1723
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1724
- [ ] https://github.com/tenstorrent/tt-mlir/issues/1725

tt-xla
- [ ] https://github.com/tenstorrent/tt-xla/issues/150
- [x] https://github.com/tenstorrent/tt-xla/issues/152

tt-torch
- [ ] https://github.com/tenstorrent/tt-torch/issues/161

**Comment by tapspatel** [comment]
Closing issue. Golden Demo will be a condensed version of all this support
